terraform/modules/dbx_uc_create_catalog/variables.tf

```diff
@@ -0,0 +1,19 @@
+variable "gcs_bucket_name" {
+  description = "Path to the external volume to be registered in the metastore"
```

Comment:
Er det noe sted vi ikke bruker gs://path/to/volume, altså inkludert gs:// ? Hvis ikke kan vi kanskje bare ta det med overalt? Eventuelt så kan vi gå for løsningen som jeg foreslo over, at bøtta opprettes i den modulen, også kan man ta bøtte gs-url som output

Reply:
Bruker ikke det i storage_bucket_iam_member.